WebApp/Views/AddOpcServer/Index.cshtml

```diff
@@ -99,8 +111,17 @@
             $("#waitingForConnect").hide();
             $("#addOpcServerBodyContextPanel").show();
             $("#addOpcServerContextpanelHeader").text("Security Warning");
```

Comment:
Needs "Security Warning" to be localized? There is also a text a bit earlier: "UntrustedCertificate" do we need to localize it too? Pls open bugs if this is the case.